src/components/FullPageWebView/FullPageWebView.tsx

```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
     }, [navigation, params.loggedIn, params.title] )
   );
 
+  const fontScalingJS
+  = `
+    if( ${isLargeFontScale} ){
+      document.body.style.fontSize = '40px';
+      let paragraphs = document.querySelectorAll('p');
+      paragraphs.forEach(p => {
+        p.style.fontSize = '50px';
+      })
+    }
+    true; // note: this is required, or you'll sometimes get silent failures
```

Comment:
What kind of errors?

Reply:
Haven't experienced any errors personally but it was listed in the docs to make sure the string evaluates to a valid type